course_catalog/task_helpers.py

```diff
@@ -459,3 +463,22 @@ def get_course_availability(course):
         for run in course_runs:
             if run.get("key") == course.course_id:
                 return run.get("availability")
+
+
+def should_skip_course(course_title):
+    """
+    Returns True if '[delete]', 'delete ' (note the ending space character)
+    exists in a course's title or if the course title equals 'delete' for the
+    purpose of skipping the course
+
+    Args:
+        course_title (str): The course.title of the course
+
+    Returns:
+        bool
+
+    """
+    title = course_title.lower()
+    if ("[delete]" in title) or ("delete " in title) or (title == "delete"):
```

Comment:
This works perfectly with the current data set.  But is it possible that in the future there may be a course with a slightly different title like "(Delete)"?  Or maybe a valid course with a title like "[Delete hidden viruses](https://www.udemy.com/how-to-delete-hidden-virus/)"?  @pdpinch if you think that is unlikely then I'll approve this PR.

Reply:
I'd be happy to change the rules as you see fit, including adding `(delete)` and/or a more broad clause of `"delete" in title`. I can also add a whitespace strip `.strip()` to strip whitespace at the ends for edge cases that may include that.

Reply:
Honestly, it's impossible to predict. I don't really know why these courses appear in the feed at all. Does this implementation catch the course whose title starts with "[DELETE]"?

Reply:
It does. It's also one of the test cases.

Reply:
A note on my decision: I'm opting to lean towards conservatism. I looked into this and the substring "delete" doesn't really show up elsewhere in English language other than in words related to or with the suffix of "delete". Despite that, I wanted to avoid using the board clause of "delete" in string, which would look for the substring "delete" anywhere in the title string. That is, it would match "deleterious" if that word happened to be in the title. So we have a rule that mostly captures the only cases we're seeing and variations on them.

Reply:
Leaning towards conservatism is the right choice. 👍